src/main/java/org/usfirst/frc/team1758/robot/commands/TurnRight.java

```diff
@@ -15,12 +15,12 @@ protected void initialize() {
 	}
 
 	protected void execute() {
-		if(sensors.getGyroAngle() < -30)
+		if(sensors.getGyroAngle() > 30)
 		{
 			finished = true;
 			driveTrain.mecanumDriveCartesian(0, 0, 0, 0);
 		} else {
-			driveTrain.tankDrive(-.75, 0);
+			driveTrain.tankDrive(.5,.5);
```

Comment:
Don't we only need one set to turn? also why is one of them not negative? shouldn't it be? If it isn't something is wrong in driveTrain.